downloads/id/github-git-cheat-sheet.md

```diff
@@ -0,0 +1,215 @@
+---
+layout: cheat-sheet
+title: Buku Saku Git GitHub
+byline: Git adalah sistem manajemen revisi terdistribusi bersifat terbuka yang menyediakan aktivitas GitHub di laptop atau komputer pribadi Anda. Buku saku ini meringkas baris perintah instruksi-instruksi Git yang paling sering dipakai sebagai referensi singkat.
+leadingpath: ../
+---
+
+{% capture colOne %}
+## Pasang Git
```

Comment:
There should be a blank line here , after the title.  Please run a markdown linter over your proposed contribution.